src/view/screens/Search/Search.tsx

```diff
@@ -347,16 +412,11 @@ export function SearchScreenInner({
           <CenteredView
             sideBorders
             style={[pal.border, pal.view, styles.tabBarContainer]}>
-            <TabBar items={SECTIONS_LOGGEDIN} {...props} />
+            <TabBar items={sections.map(section => section.title)} {...props} />
           </CenteredView>
         )}
         initialPage={0}>
-        <View>
-          <SearchScreenPostResults query={query} />
-        </View>
-        <View>
-          <SearchScreenUserResults query={query} />
-        </View>
+        {sections.map(section => section.component)}
```

Comment:
According to the docs, its direct children have to be `View`s: https://github.com/callstack/react-native-pager-view?tab=readme-ov-file#usage